docs/bitcoin/mempool.md

```diff
@@ -1 +1,15 @@
 # Mempool Space
+
+## Overview
+Bitcoin users broadcast their transactions via a node. Miners take these transactions, verify them and add them to the blockchain for the coinbase reward. Since the blocksize is limited, if there are more transactions than the blocksize, they must wait in buffer space. This buffer space is called the **mempool**. The size of the mempool indicates the demand for space on the bitcoin blockchain and directly affects the fees required to confirm transactions.
+
+Mempool.space is a visualization software for the bitcoin mempool. It decomposes the mempool based on transaction fees which helps in deciding fees. It also acts as an explorer for transactions and public addresses.
+
+Mynode implementation of mempool.space is limited by default. The mempool size is capped at 50MB to limit RAM usage. One can change that in the `bitcoin.conf`. Mempool.space can be enabled from the homepage and used at port 4080/4081. Initially, it takes a few minutes to load first few blocks and the graph for last 2 hours. Also, the mynode implementation differ from that on official website of [mempool.space](https://mempool.space). v1 included in mynode is compatible with single-board computers like Raspberry Pi whereas the latest v2 is not.
```

Comment:
The myNode implementation of mempool.space is limited by default. The mempool size is capped at 50MB to limit RAM usage. One can change that limit in the `bitcoin.conf`file, which is accessed by clicking the "Manage" button under Bitcoin on the homepage. Mempool.space can be enabled from the homepage and used at port 4080/4081. Initially, it takes a few minutes to load first few blocks and a graph for the most recent 2 hours of activity. Also, the myNode implementation differs from that on the official website of [mempool.space](https://mempool.space). v1 included in myNode is compatible with single-board computers like Raspberry Pi whereas the latest v2 is not.

Reply:
added